File: frontend/vue/features/vue-java/skeleton/server/src/main/java/@defaultPackage.path@/ApplicationController.java
@defaultPackage@;

import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;


@Controller("/application")
public class ApplicationController {

    @Get("/")
    Map index() {
        return [version: '0.1', name: '@appName@'];
    }
}

File: frontend/vue/features/vue-kotlin/skeleton/server/src/main/kotlin/@defaultPackage.path@/ApplicationController.kt
package @defaultPackage@

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/application")
class ApplicationController {

    @Get("/")
    fun index(): Map {
        return [micronautVersion: '@version@', name: '@app.name@']
    }
}